src/main/java/uk/gov/hmcts/reform/pip/data/management/models/publication/ListType.java

```diff
@@ -20,7 +20,8 @@ public enum ListType {
     FAMILY_DAILY_CAUSE_LIST(LocationType.VENUE),
     CIVIL_AND_FAMILY_DAILY_CAUSE_LIST(LocationType.VENUE),
     COP_DAILY_CAUSE_LIST(LocationType.VENUE),
-    SSCS_DAILY_LIST(LocationType.REGION);
+    SSCS_DAILY_LIST(LocationType.REGION),
+    PRIMARY_HEALTH_LIST(LocationType.NATIONAL);
```

Comment:
Not for this repo, but we'll need equivalent ones in the others too for the new list type